lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -189,7 +190,7 @@ class CategoryRenderer {
     const notApplicableElem = this.renderAuditGroup({
       title: `Not applicable`,
     }, {expandable: true, itemCount: this._getTotalAuditsLength(elements)});
-    notApplicableElem.classList.add('lh-audit-group--notapplicable');
+    notApplicableElem.classList.add('lh-audit-group--adorned', 'lh-audit-group--notapplicable');
```

Comment:
a11y audits are the only unadorned audit-groups now then?

Reply:
a11y and seo audit groups don't get icons, yeah.